python/pyspark/pandas/namespace.py

```diff
@@ -272,7 +272,7 @@ def read_csv(
         The character used to denote the start and end of a quoted item. Quoted items can include
         the delimiter and it will be ignored.
     escapechar : str (length 1), default None
-        One-character string used to escape delimiter
+        One-character string used to escape other characters.
```

Comment:
Would it be more accurate to say "delimiters and quotes"? not all other chars are escaped of course

Reply:
Happy to merge this with this change, maybe?